azkaban-common/src/main/java/azkaban/database/DataSourceUtils.java

```diff
@@ -129,11 +129,13 @@ private DataSourceUtils() {
 
     private static MonitorThread monitorThread = null;
 
+    private final String url;
```

Comment:
I wanted to log the url in case of DB connection failures. 
Apparently there is no easy API to access the url from that class.